app/src/test/java/ch/epfl/qedit/model/UserTest.java

```diff
@@ -19,4 +21,30 @@ public void dummyTestUser() {
         assertTrue(user.equals(user));
         assertFalse(user.equals(new User("Bill", "Gates", User.Role.Participant)));
     }
+
+    @Test
+    public void quizzesTest() {
+        User user = new User("John", "Doe", User.Role.Administrator);
+        user.addQuiz("q0", "First Quiz");
+        user.addQuiz("q1", "Second Quiz");
+        assertThat(user.getQuizzesKey(), hasItems("q0", "q1"));
+        assertThat(user.getQuizzesTitle(), hasItems("First Quiz", "Second Quiz"));
+    }
+
+    @Test
+    public void addQuizzesAddlTest() {
```

Comment:
Addl?

Reply:
I honestly do not know what the Addl stands for but following the naming of the other tests I'm going to replace it by: addQuizzesTest.